vars/abapEnvironmentPipelineStagePrepareSystem.groovy

```diff
@@ -0,0 +1,24 @@
+import com.cloudbees.groovy.cps.NonCPS
+import com.sap.piper.GenerateStageDocumentation
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field STAGE_STEP_KEYS = []
+@Field Set STEP_CONFIG_KEYS = []
+@Field Set PARAMETER_KEYS = []
+/**
+ * This stage prepares the SAP Cloud Platform ABAP Environment systems
+ */
+void call(Map parameters = [:]) {
+    def script = checkScript(this, parameters) ?: this
+
+    def stageName = parameters.stageName?:env.STAGE_NAME
+
+    cloudFoundryCreateService script: parameters.script
+    input message: "Steampunk system ready?"
```

Comment:
Since the full Steampunk provisioning takes langer than the actual CF Service Creation I would propose to inform the user that systems might still be in an usable state due to provisioning or simply let the user check if system is ready.

Suggestion: `input message: "Steampunk system ready? Please make sure that the system provisioning is fully completed before creating the Communication Arrangement 0510."`